src/WordPress_Object/Clarkson_Object.php

```diff
@@ -67,7 +69,7 @@ public function __get( $name ) {
 	 *
 	 * @return \WP_Post The post object.
 	 */
-	public function get_object(): \WP_Post {
```

Comment:
Deze `get_object()` moet nog wel blijven bestaan, maar is deprecated. Gaat er dan in de volgende major release uit.